### PR TITLE
New package: wmix-3.5

### PR DESCRIPTION
**Pull request by lordfeck**
ALSA sound controller dockapp. Tried in both Window Maker and Fluxbox, should work with any window manager. Supports channel selection and muting too. Its appearance is in the dockapp vernacular of late 90s home appliances.

Also catches the volume controls on the keyboard.

### CODE REVIEW

File: srcpkgs/wmix/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'wmix'
+pkgname=wmix
+version=3.5
+revision=1
+build_style=gnu-configure
+makedepends="alsa-lib-devel libXpm-devel libXrandr-devel"
+short_desc="Dockapp to control ALSA sound. Channel selector and keyboard support"
+maintainer="lordfeck <bs+dev@thran.uk>"
+license="GPL-2.0-only"
+homepage="https://www.dockapps.net/category/audio"
```
Comment (sgn) [inline srcpkgs/wmix/template:10]: I prefer https://www.dockapps.net/wmix for `homepage`

File: srcpkgs/wmix/template [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'wmix'
+pkgname=wmix
+version=3.5
+revision=1
+build_style=gnu-configure
+makedepends="alsa-lib-devel libXpm-devel libXrandr-devel"
+short_desc="Dockapp to control ALSA sound. Channel selector and keyboard support"
```
Comment (sgn) [inline srcpkgs/wmix/template:7]: I think better `short_desc` is "WindowMaker's dockapp mixer for OSS or ALSA"

**Comment by leahneukirchen** [comment]
Note we also have `AlsaMixer.app` already.

File: srcpkgs/wmix/template (hunk 2 above)
Comment (lordfeck) [reply srcpkgs/wmix/template:7]: Will make those changes and push shortly.

**Comment by lordfeck** [comment]
@leahneukirchen I've tried AlsaMixer.app, however [wmix is much more extensive in its feature support](https://www.dockapps.net/wmix). It allows one to adjust any of the configured ALSA channels with and easy switcher. Also looks a whole lot cooler, in my view.

**Comment by abenson** [comment]
This needs to be squashed down to a single `New package: wmix-3.5` commit

**Comment by lordfeck** [comment]
@abenson @sgn Ah shite, looks like the repo has been thrashed in my process of trying to squash then push.

Right this might be a hard fix. I'll abandon this PR and start afresh tomorrow, if everyone doesn't mind.

**Comment by lordfeck** [comment]
Hate to bother everyone again, but I've opened a new PR. I think it was easier than trying to reconcile the broken repo.
This time I've used a branch, so it should work properly.

Link to the new PR: https://github.com/void-linux/void-packages/pull/27278

I'd appreciate any review/comments there. Thanks.

**Pull request closed without merging**